build/regressions.config.yml

```diff
@@ -85,6 +85,7 @@ sources:
         - area-perf
       owners:
         - sebastienros
+        - eerhardt
```

Comment:
You can add yourself only for Stage1Aot (same for labels)

```yaml
    rules:
      - include: FortunesEF
      - include: Stage1Aot
        owners:
          - eerhardt
```

Reply:
Does this sample count as documentation?

https://github.com/dotnet/crank/blob/main/src/Microsoft.Crank.RegressionBot/sample.config.yml#L7-L14

Reply:
Done.